less/components/_buttons.less

```diff
@@ -20,6 +20,10 @@
   padding: 1rem 3.5rem;
 }
 
+.btn-xs {
+  padding: 0.1rem 0.5rem;
```

Comment:
While you are fixing .btn-xs, I suspect that the other buttons size are not working as expected because of 

.btn {
  padding: 0.8rem 1.5rem;
}
The bootstrap project utilizes variables and the .button-size mixing to set the padding
https://github.com/twbs/bootstrap/blob/v3-dev/less/buttons.less

Reply:
I didn't want to touch those because other components also use `@padding-base-vertical` and the other variables. There isn't a variable dedicated to buttons in BS3.

Reply:
You should be able to search for the less code and confirm that.  My main concern here is that we are overriding the padding with .btn. which is overriding all the .btn size classes to use the same padding. I suspect that .btn-lg and .btn-sm  are not working as expected anymore.

Reply:
Added an MR for EFSA to add dedicated button variables: https://gitlab.eclipse.org/eclipsefdn/it/webdev/solstice-assets/-/merge_requests/473